nutkit/frontend/config.py

```diff
@@ -0,0 +1,27 @@
+from dataclasses import dataclass
+from typing import (
+    Callable,
+    Dict,
+    List,
+    Optional,
+)
+
+
+@dataclass
+class Neo4jBookmarkManagerConfig:
+    initial_bookmarks: Optional[Dict[str, List[str]]] = None
+    bookmark_supplier: Optional[Callable[[str], List[str]]] = None
+    notify_bookmarks: Optional[Callable[[str, str], None]] = None
+
+
+def to_protocol(
```

Comment:
Maybe this should be a method of `Neo4jBookmarkManagerConfig` instead? I'm thinking of how to add more config classes in the futuer.